app/helpers/image_service_helper.rb

```diff
@@ -82,13 +84,19 @@ def member_image_tag(parent_id:, member:, size_key: nil, lazy: false, use_image_
 
   private
 
-  def create_riiif_url(path)
-    if CHF::Env.lookup(:public_riiif_url)
-      url = Addressable::URI.parse(CHF::Env.lookup(:public_riiif_url))
-      raise "public_riiif_url requires a valid URL with host, eg `http://host` or `//host`" if url.host.nil?
-      return Addressable::URI.join(url, path).to_s
-    else
-      return path
+  attr_accessor :public_host
+
```

Comment:
what is an attr_accessor on a Helper module supposed to do? I think it's unneccesary and may do weird things?